vite-js/basics/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "vite": "^6.0.11"
+  }
+}
```

Comment:
how do we run this? tried `npm run dev` `npm start` `node .` `node vite` `vite`...

Reply:
better to have a run script here, and a `Makefile` for me :pray: